main.tf

```diff
@@ -81,7 +80,7 @@ data "google_container_engine_versions" "gke-version" {
 
 module "gke" {
   source                        = "terraform-google-modules/kubernetes-engine/google//modules/private-cluster"
-  version                       = "14.3.0"
+  version                       = "15.0.2"
```

Comment:
No breaking changes here? Just asking as it's a major jump from 14 -> 15.

Reply:
None that I found affects us, from my change notes above I reviewed the 15.0.2 changelog and there was no big changes in modules we use.
https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/v15.0.2/CHANGELOG.md

Reply:
Here the list of [breaking changes](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/releases/tag/v15.0.0). You have to look at the initial major version update for breaking changes. Please verify

Reply:
@thpang See my my previous comment. It has that same link, already made sure that any breaking changes from 15.0.0-15.0.2 didn't require us to make any changes